docs/core/extensions/windows-service.md

```diff
@@ -149,9 +149,16 @@ For more information, see [`dotnet publish`](../tools/dotnet-publish.md).
 To create the Windows Service, use the native Windows Service Control Manager's (sc.exe) create command. Run PowerShell as an Administrator.
 
 ```powershell
-sc.exe create ".NET Joke Service" binpath=C:\Path\To\App.WindowsService.exe
+sc.exe create ".NET Joke Service" binpath="C:\Path\To\App.WindowsService.exe"
 ```
 
+> [!TIP]
+> If you need to change the content root, you can pass it as a command-line argument when specifying the `binpath`:
```

Comment:
Since *content root* isn't mentioned elsewhere in this article, it might be worth making it a link to an article that explains the term.

Reply:
```suggestion
> If you need to change the content root of the [host configuration](../generic-host.md#host-configuration), you can pass it as a command-line argument when specifying the `binpath`:
```